aes/src/lib.rs

```diff
@@ -115,6 +115,7 @@
 )]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
 #![warn(missing_docs, rust_2018_idioms)]
+#![allow(unsafe_op_in_unsafe_fn)]
```

Comment:
I think it's better to do this `allow` on the per-module basis like I did in the hashes PR. With potential `target_feature` improvements (which would make some intrinsics safe inside functions with with enabled target features) we may enable this lint back.

Reply:
Ha, I've tried to read the patch for hashes, but github UI is very slow. I've thought you've used a big `unsafe{` block, but that was only in `sha1`, I should have kept reading, my bad.

Reply:
I did it in the initial commits, but then switched to `#[allow(..)]`. For example [here](https://github.com/RustCrypto/hashes/blob/cfe82a0f1a9a6c89c5b6d8309f2603c644dc97ea/sha1/src/compress/x86.rs#L34) I enable it for the function and [here](https://github.com/RustCrypto/hashes/blob/cfe82a0f1a9a6c89c5b6d8309f2603c644dc97ea/sha2/src/sha256/aarch64.rs) for the whole module.